hail/src/test/scala/is/hail/expr/ir/EmitStreamSuite.scala

```diff
@@ -128,48 +131,6 @@ class EmitStreamSuite extends HailSuite {
     }
   }
 
-  @Test def testES2LeftJoinDistinct() {
```

Comment:
So I moved some Stream operations from Stream to EmitStream and made them Stream[EmitCode]-specific.  Those functions are directly used in the implementation of IR nodes that have their own tests (e.g. IRSuite.testLeftJoinRightDistinct).  Therefore, I was OK dropping them rather than writing them in terms of EmitCode.